pycls/core/config.py

```diff
@@ -205,6 +205,8 @@
 # Type of classifier select from {'token', 'pooled'}
 _C.VIT.CLASSIFIER_TYPE = "token"
 
+# TODO(anj): Debug error: https://www.internalfb.com/phabricator/paste/view/P479199402
```

Comment:
This might get fixed if you rebase to the latest PyCls main.

Reply:
Looks like you did a rebase to main on my branch? I'll retest it and see if it works.